packages/documentsearch/style/base.css

```diff
@@ -34,11 +34,15 @@
   outline: 0;
 }
 
-.jp-DocumentSearch-overlay button:hover {
+.jp-DocumentSearch-overlay button.jp-DocumentSearch-button-wrapper:disabled {
+  opacity: 0.6;
```

Comment:
When I hover over the debugger button when disabled it shows not allowed cursor. When I hover over the disable search buttons they show hand cursor. Should we align to the debugger button style?

```suggestion
  opacity: 0.6;
  cursor: not-allowed;
```

![cursor-style](https://github.com/jupyterlab/jupyterlab/assets/5832902/2e4461c7-aca2-4329-9cce-20f2a4feb97e)

Reply:
Also debugger button uses 0.4 opacity (I do not have a preference here).

Reply:
I don't see the pointer cursor on the prev/next buttons in Firefox ESR, and the `cursor: not-allowed` rule doesn't seem to change the cursor to the 🚫 cursor as your animation shows.